index.html

```diff
@@ -412,12 +412,31 @@
     </section>
     <section>
       <h2>
+        Navigator and WorkerNavigator extension
+      </h2>
+      <p>
+        A <a>Permissions</a> instance is exposed on the <code>navigator</code>
+        object for <code>Window</code> and <code>Worker</code> contexts.
+      </p>
+      <dl class='idl' title='[Exposed=(Window)] partial interface Navigator'>
```

Comment:
sorry, why do you have two definitions for Exposed? Don't you just want one, with `[Expose=(Window,Worker)]`?